_docs/README.md

```diff
@@ -153,7 +156,8 @@ boilerplate --template-folder ~/templates --output-folder ~/output --var-file va
 #### The boilerplate.yml file
 
 The `boilerplate.yml` file is used to configure `boilerplate`. The file is optional. If you don't specify it, you can
-still use Go templating in your templates, but no variables or dependencies will be available.
+still use Go templating in your templates so long as you specify the `--missing-config-action ignore` option, but no
+variables or dependencies will be available.
```

Comment:
Once we have plugins, sure, there may be other cases where a `boilerplate.yml` file is not necessary. If that turns out to be a common case, we can even change the default value to `ignore`. However, as it is now, I had a few cases where I had large `boilerplate.yml` files, with many dependencies, and a few of the dependencies had typos in their template folder paths. I didn't notice this because boilerplate just logged a warning.